src/xrpld/rpc/handlers/Tx.cpp

```diff
@@ -100,7 +100,6 @@ doTxPostgres(RPC::Context& context, TxArgs const& args)
     if (locator.isFound())
     {
         auto start = std::chrono::system_clock::now();
-        // The second argument of fetch is ignored when not using shards
         if (auto obj = context.app.getNodeFamily().db().fetchNodeObject(
                 locator.getNodestoreHash(), locator.getLedgerSequence()))
```

Comment:
> The second argument of fetch is ignored when not using shards

So does it make sense to remove the second argument to `fetchNodeObject` everywhere?

Reply:
It does. I started going down that route, but it is all over the place, and I started having second thoughts. We might want to keep this information available for the future. It seems to be a lesson learned the hard way that we often want _both_ the ledger digest _and_ the ledger sequence together. In fact, I added a type in my FLR2 branch just for representing the pair, which I call `LedgerIdentifier`.

Reply:
That ledger sequence has a default value of 0. Do you know if there are any places that are using the default, or passing 0 in explicitly? If there aren't any, then I'm fine with leaving this as-is.

Reply:
I know there are places that pass 0 explicitly. Here is [one](https://github.com/XRPLF/rippled/blob/8b181ed81883ce9f0a1c54f8bad9347ac7066627/src/xrpld/app/ledger/detail/SkipListAcquire.cpp#L80) I found (in FLR1) in a quick search. I believe there are more. 0 is not a valid sequence though. The genesis ledger [starts](https://github.com/XRPLF/rippled/blob/8b181ed81883ce9f0a1c54f8bad9347ac7066627/src/xrpld/app/ledger/Ledger.cpp#L190) at 1. 0 is used to mean "unknown".

Reply:
Yeah, that's what I figured.